frontend/packages/console-app/console-extensions.json

```
@@ -645,17 +645,7 @@
       "perspective": "admin",
       "section": "ecosystem",
       "name": "%console-app~Ecosystem%",
-      "insertBefore": "operators"
-    }
-  },
-  {
-    "type": "console.navigation/section",
-    "properties": {
-      "perspective": "admin",
-      "id": "operators",
-      "name": "%console-app~Operators%",
-      "dataAttributes": { "data-quickstart-id": "qs-nav-operators" },
-      "insertAfter": "home"
+      "insertAfter": "operators"
```

Add quickstart-id here

```suggestion
      "insertAfter": "operators",
      "dataAttributes": { "data-quickstart-id": "qs-nav-ecosystem" },
```

---

Thanks for the reviews. i made those changes and was wondering how to test web-terminal-adminuser.ts locally